bodhi/tests/server/tasks/test_updates.py

```diff
@@ -23,7 +23,9 @@
 from urllib.error import URLError
 
 import sqlalchemy
+from fedora_messaging import testing as fml_testing
```

Comment:
Just out of curiosity, how is `fedora_messaging` shortened to `fml`, where does the `l` come from? Is this an inside joke? :wink:

Reply:
I think it is acronym for Fedora Messaging Library, but I'm not sure.

Reply:
No clue, this was just some copy/pasta :-)